src/ui/UserProfile.h

```diff
@@ -8,15 +8,17 @@
 class DeviceInfo
 {
 public:
-        DeviceInfo(const QString deviceID, const QString displayName)
+        DeviceInfo(const QString deviceID, const QString displayName, const bool verified)
```

Comment:
Don't make values const, if you copy them. Either take them by const& or just copy them, if that is cheaper. Concretely, you should remove the const here from bool. Not so sure about QString. QString is copy on write, a const& forces a deep copy though. I would probably use a copy here instead of a const&.